bittide/src/Bittide/ClockControl/Callisto.hs

```diff
@@ -167,8 +167,8 @@ callisto ControlConfig{..} mask scs dataCounts state =
   --
   -- the constants here are chosen to match the above code.
```

Comment:
We should update this comment that we copied the proportional gain _k_ from the Julia code, but that we set the step size to match the step size for which the PLL is configured. Perhaps commending on why we kick 100e-9 instead of 5e-7; I assume this might have something to do with the PLLs in the CDRs on the receive side losing track if we were to make the step size that large?

Reply:
Agreed